LOG.tbasicu

```diff
@@ -150,33 +155,32 @@ Type tLogThis
     Local sTagF as String
     Local sDT as String
     Local sDTF as String
-    Local successFile as Number
-    Local logWhatTitleFile = logWhatTitle
+    Local logWhatTitleFile as String = logWhatTitle
     
-    '[t] Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 ) ...
-    local TagDebug = me.TagDebug 
-    local TagInfo = me.TagInfo
-    local TagNotice = me.TagNotice
-    local TagWarn = me.TagWarn
-    local TagErr = me.TagErr
-    local TagCrit = me.TagCrit
-    local TagAlert = me.TagAlert
-    local TagPanic = me.TagPanic
+    #region "Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 )" '[t]
+    local TagDebug as String = me.TagDebug 
+    local TagInfo as String = me.TagInfo
+    local TagNotice as String = me.TagNotice
+    local TagWarn as String = me.TagWarn
+    local TagErr as String = me.TagErr
+    local TagCrit as String = me.TagCrit
+    local TagAlert as String = me.TagAlert
+    local TagPanic as String = me.TagPanic
     
-    Local ColorDebugText = me.ColorDebugText
-    Local ColorBackground = me.ColorBackground
-    Local ColorText = me.ColorText
-    ' ... workaround
+    Local ColorDebugText As Long = me.ColorDebugText
+    Local ColorBackground As long = me.ColorBackground
+    Local ColorText As Long = me.ColorText
+    #endregion
     
-    '[t] Workaround for PrintL error ( see https://www.thinbasic.com/community/project.php?issueid=617 ) ...
+    #region "Workaround for PrintL error ( see https://www.thinbasic.com/community/project.php?issueid=617 )" '[t]
     Local myColor as number
-    ' ... workaround
+    #endregion
     
-    Local logFile = me.logFile
+    Local logFile as string = me.logFile
     
     if me.isDebug or not Bit_Get(logWhere,0) then ' Display debug and production messages (all messages) OR Display production messages only (no debug messages)
       
-      if Bit_Get(logWhere,1) then ' %LT_Time is SET, adds timestamp
+      if me.private_isFlagSet(logWhere, %LT_Time) then ' adds timestamp
```

Comment:
Sebastian,

I hope you will like this improvement in readability.

You no longer need to put the %LT_TIME in comment + you completely eliminate risk of using wrong number.

I found such error on one place, now it is gone thanks to these changes.

Reply:
indeed, I like it